packages/web-components/src/components/rux-input/rux-input.tsx

```diff
@@ -297,6 +298,7 @@ export class RuxInput implements FormFieldInterface {
                     ) : null}
 
                     <div
+                        part="input-field"
```

Comment:
I almost feel like `input-wrapper` would be a better name? I care like a 3/10 though so this is non-blocking. @markacianfrani , whatdya think

Reply:
I was oscillating between wrapper and field. I would also normally gravitate toward wrapper but I went into some of our other components for inspiration and noticed that others were using -field to denote something wrapping a form item. Of course in those cases, the field usually includes the label. I was aiming at consistency but am happy to change it! :)

Reply:
I'd go for consistency in this case. This one is difficult. I think 'wrapper' as a term is semantic junk food, but I still use it all the time. The difference between a part and a class though is we need to think about support. We can change the internals (add more wrappers inside wrappers), but we always need to keep this part so it needs to have a more semantic meaning. Having said that . . . I dont have any better semantic names in mind for this particular one